test/problems/multipass/output_validators/multipass_validator/multipass_validator.py

```diff
@@ -2,18 +2,36 @@
 import sys
 import pathlib
 
+judgemessage = pathlib.Path(sys.argv[3]) / 'judgemessage.txt'
 test_in = int(pathlib.Path(sys.argv[1]).read_text())
-team_ans = int(input())
+try:
+    team_ans_string = input()
+    try:
+        team_ans = int(team_ans_string)
+    except ValueError:
+        judgemessage.write_text(f"team output '{team_ans_string}' is not an integer")
+        sys.exit(43)  # WA
```

Comment:
could you wrap the `judgemessage.write_text` and `exit(43)` together in a function?

Reply:
Done